pandas/tests/copy_view/test_indexing.py

```diff
@@ -318,7 +326,8 @@ def test_subset_set_with_row_indexer(backend, indexer_si, indexer, using_copy_on
     ):
         pytest.skip("setitem with labels selects on columns")
 
-    if using_copy_on_write:
+    # TODO
```

Comment:
This should warn, shouldn't it?

Reply:
talking about the regular CoW mode

Reply:
No, for the regular CoW mode is shouldn't warn I think. The test is a bit hard to read, but it boils down to:

```
df = DataFrame({"a": [1, 2, 3, 4], "b": [4, 5, 6, 7], "c": [0.1, 0.2, 0.3, 0.4]})
subset = df[1:4]

subset.iloc[0:2] = 0
```

So this is a case that should warn in the warn mode (because it currently _does_ modify `df`), but it's not chained assignment, so the standard CoW rules apply and `df` doesn't get updated